### ISSUE TITLE
Kissmanga cannot download and see chapter

### ISSUE THREAD

**Issue by zartzaku** [riderkick/FMD#1705]
Just today i check i cannot download from kissmanga. the manga chapter not loaded and show nothing. been trying to use this trick (https://github.com/riderkick/FMD/wiki/FAQ#cannot-download-from-kissmanga) but still not work.

**Comment by echoPN** [comment]
I have the same problem...
Someone Fix it, please...

**Comment by echoPN** [comment]
I tried using this trick...

https://github.com/riderkick/FMD/issues/1699

and it's work...
:v /

**Comment by echoPN** [comment]
I tried using this trick...

https://github.com/riderkick/FMD/issues/1699#issuecomment-474183523_

and it's work...
:v /

**Comment by traitorAIZEN** [comment]
That also worked for me but ther was a bit deviation from #4 instruction.
In my case there's no need to go to the "Cookies" tab. The cookie to be copied is already somewhere below the user-agent in the headers tab, no more need to copy individually since it's all in a one line.

**Comment by VijayThagriya** [comment]
Tried Both the method, worked non of them.

**Comment by AizenOusen** [comment]
not working for me too. please fix.

**Comment by BluePlasticCup** [comment]
echoPN's link worked and LN sensei traitorAIZEN's step saved me some time.

Thank you both.

**Comment by zartzaku** [comment]
echoPN solution worked, thank you.

**Comment by Tmp341** [comment]
I've added that workaround to [FAQ](https://github.com/riderkick/FMD/wiki/FAQ/fceef23ebb4b775fab6909eab72915541024eded#cannot-download-from-mangadexkissmanga).

**Comment by VijayThagriya** [comment]
> I've added that workaround to [FAQ](https://github.com/riderkick/FMD/wiki/FAQ/fceef23ebb4b775fab6909eab72915541024eded#cannot-download-from-mangadexkissmanga).

Thanks for Help. But i am not able to find Cookies tab next to the header in Chrome BRower.

ANd again both method not working for me . After Updating it to .158

**Comment by AizenOusen** [comment]
i have done both still no result

**Comment by zartzaku** [comment]
today, the chapter shows, but cannot downloaded. always fail. only show 1 page.

**Comment by Slasar41** [comment]
Working for me without changing all the "Max-" and "UpdateList-" options to 1.

**Comment by mardisyahmani** [comment]
changing all the "Max-" and "UpdateList-" options to 1 not work for me.
When I change it back to 0 it's working fine.

**Comment by traitorAIZEN** [comment]
I got the same problem again today, then i just redid the steps and is now working again. No more need to change the 'Max' options

**Comment by srishak34** [comment]
for anyone using the chrome browser, i did it exactly as the instruction said and it worked, to go to "Cookies" open the developer tools -> Application -> then on the list there is Cookies section. use the filter to search cfduid, cfclearance, and idtz and set all the "Max-" and "UpdateList-" options to 1 then apply (for me, the application crash one time then the next works just fine but had to do it all over again. idk if this is caused by me doing something wrong but it works). this worked on 23 march 2019

**Comment by AizenOusen** [comment]
worked thanks.

**Comment by VijayThagriya** [comment]
> for anyone using the chrome browser, i did it exactly as the instruction said and it worked, to go to "Cookies" open the developer tools -> Application -> then on the list there is Cookies section. use the filter to search cfduid, cfclearance, and idtz and set all the "Max-" and "UpdateList-" options to 1 then apply (for me, the application crash one time then the next works just fine but had to do it all over again. idk if this is caused by me doing something wrong but it works). this worked on 23 march 2019

Thank you very much bud.

**Comment by Tmp341** [comment]
I've tried to check my favorites (i've added cookie 2 days ago). It looks like, you have to change the cookie everytime you want to use (ie until your cookie expires) Kissmanga. Im not a coding expert, so if you know a short way, you can get that cookie, and replace it in``Config\modules.json`` file.

**Comment by MasuRii** [comment]
> I've tried to check my favorites (i've added cookie 2 days ago). It looks like, you have to change the cookie everytime you want to use (ie until your cookie expires) Kissmanga. Im not a coding expert, so if you know a short way, you can get that cookie, and replace it in`Config\modules.json` file.

Yes the cookie Expires but there's a trick to it you just need to go Kissmanga and let it open there and leave it so that the cookie won't expire and don't close the browser.. I tried this already and it works

**Comment by BluePlasticCup** [comment]
Stopped working a few hours ago. Had to change the UserAgent after a Firefox update. Works again.

**Comment by ayanamichie** [comment]
Disabled google DCP, 'Max' and 'Update' set out to 0, got only Cookie and UserAgent, and its working.
Did use same method to get those as AIZEN.

**Comment by forvy** [comment]
Not working using Chrome or Firefox. Have followed [FAQ](https://github.com/riderkick/FMD/wiki/FAQ/fceef23ebb4b775fab6909eab72915541024eded#cannot-download-from-mangadexkissmanga)

Resulting on Just a Moment in Manga Info and failure in favorite update check.

**Comment by VijayThagriya** [comment]
> Not working using Chrome or Firefox. Have followed [FAQ](https://github.com/riderkick/FMD/wiki/FAQ/fceef23ebb4b775fab6909eab72915541024eded#cannot-download-from-mangadexkissmanga)
> 
> Resulting on Just a Moment in Manga Info and failure in favorite update check.

Yes its not working Any longer

**Comment by Slasar41** [comment]
> > Not working using Chrome or Firefox. Have followed [FAQ](https://github.com/riderkick/FMD/wiki/FAQ/fceef23ebb4b775fab6909eab72915541024eded#cannot-download-from-mangadexkissmanga)
> > Resulting on Just a Moment in Manga Info and failure in favorite update check.
> 
> Yes its not working Any longer

It's still working.
Just now I Check for update and can download it.
Only need to change the cookies.

**Comment by Tmp341** [comment]
I've added the following sentence to the FAQ : 
> Redo this method every now and then, because cookies expire. If this is not working, it means you have to do this from the start.

**Comment by ayanamichie** [comment]
> I've added the following sentence to the FAQ :
> 
> > Redo this method every now and then, because cookies expire. If this is not working, it means you have to do this from the start.

can confirm, i did just got the cookies and working fine again.

**Comment by Owyn** [comment]
Added cookies and header but still doesn't work :-\

**Comment by Tmp341** [comment]
If anyone knows anything about coding, check this new repository [here](https://github.com/fmd-project-team/FMD).
